### DISCUSSION TITLE
Importing Bootstrap 5 into a Vue3/Vite project

### DISCUSSION THREAD

**Discussion by sunergeo** [twbs/bootstrap#36686] [Q&A]
Hello!

I have encountered problems trying to import bootstrap into a Vue3/Vite project.  I have reviewed all the discussions and repos associated with https://github.com/twbs/bootstrap/pull/36412.  I created a fork of the examples repo at https://github.com/sunergeo/bs-examples and added a vite-vue example to illustrate what I am trying to accomplish.

I am having trouble getting bootstrap to import into a basic Vue3/Vite app. I have studied the other examples in this repository, and cannot figure out what I am missing.

When the project builds, the following error is generated:

ENOENT: no such file or directory, stat '[base path]/vite-vue/node_modules/bootstrap/scss/bootstrap'

This is coming from the vite:css plugin, and I can find no information about how to configure that plugin correctly.

I have verified my vite.config.js has the correct ~bootstrap alias by hard-coding the path to node_modules/bootstrap/scss/bootstrap, and the same error occurs. If I reference the _bootstrap.scss file directly, another error occurs because the inline comments are unrecognized.

Any assistance with that issue would be greatly appreciated!

**Comment by dosstx** [comment]
Can you provide a reproduction via stackblitz.com? I'll take a look.

**Comment by sunergeo** [comment]
Thanks for the assistance!

I created a project on StackBlitz at https://vue-4u8dy7.stackblitz.io

However, the problem requires it be run in Vite for the issue to manifest.  I couldn't figure out how duplicate it on StackBlitz.

**Reply by dosstx** [reply]
For some reason, I can not get that link to open. Can you try again? Also, did you use the Vue 3 + Vite template they have on stackblitz? Be sure to select that template to get started.

**Comment by sunergeo** [comment]
Ok, I just blew away that other project and started over.  I also added you to the project as an editor.  New project is available here: https://stackblitz.com/edit/vue-5wghzw?file=package.json

For some reason, Stackblitz is complaining that it cannot find bootstrap - I'm referencing the 5.2.0-beta1 version, not sure why it doesn't like it.  Also, I cannot get Stackblitz to show the actual error I am encountering.  That only occurs when you download the full project, run yarn install and then yarn run start.  Open the project in the browser and you get the NOENT error.

Thanks again for your help!
